snuba/datasets/cdc/message_filters.py

```diff
@@ -31,9 +31,7 @@ def should_drop(self, message: Message[KafkaPayload]) -> bool:
         )
 
         if table_name:
-            table_name = table_name.decode("utf-8")
-            if table_name != self.__postgres_table:
```

Comment:
(This was causing a str/bytes mismatch on the `table_name` local variable, so I collapsed it into the conditional.)